python/lsst/meas/extensions/scarlet/multiBandPeakDetectionTask.py

```diff
@@ -0,0 +1,147 @@
+# This file is part of meas_extensions_scarlet.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+
+import lsst.afw.image as afwImage
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
+import lsst.pipe.base.connectionTypes as cT
+import lsst.scarlet.lite as scl
+
+from . import utils
+
+log = logging.getLogger(__name__)
+
+__all__ = [
+    "MultiBandPeakDetectionTask",
+    "MultiBandPeakDetectionConfig",
+    "MultiBandPeakDetectionConnections",
+]
+
+
+class MultiBandPeakDetectionConnections(
```

Comment:
Is it really not possible to consolidate this task with `MultiBandDetection`? Maybe the task and connections are trivial but some of the config fields look identical.

Reply:
I could, but they're different tasks that do slightly different things, run at different times in the same pipeline. If anything I could make `MultiBandPeakDetectionTask` a subtask of `MultiBandDetectionTask`, so that it can be run on its own before deconvolution and again after deconvolution (if necessary).

If you don't mind I'd like to keep them separate for now, since
a) I don't know if `MultiBandPeakDetectionTask` works as well as I'd hoped (right not it looks like it's having some trouble)
b) It gives me more freedom to tinker with both of them independently while I test things out.

This doesn't mean that they'll always be separate, but for now I think that it's convenient to keep them the way that they are.